tests/adapter/mongo/test_diagnosis_handler.py

```diff
@@ -124,3 +124,38 @@ def test_omim_genes(adapter, test_omim_term):
 
     # THEN it should return the right gene
     assert result[0]["hgnc_id"] == omim_gene_id
+
+
+def test_disease_terminology_count(adapter, test_omim_term, test_orpha_term):
+    # GIVEN a database with disease_terms from two coding systems
+    adapter.disease_term_collection.insert_one(test_omim_term)
+    adapter.disease_term_collection.insert_one(test_orpha_term)
+
+    # WHEN the database is queried for the disease_term counts
+    result = list(adapter.disease_terminology_count())
+
+    # THEN it should return the correct counts for each terminology
+
+    assert {"_id": "OMIM", "count": 1} in result
+    assert {"_id": "ORPHA", "count": 1} in result
+
+
+@pytest.mark.parametrize(
+    ("query", "source"),
+    [("defi", "OMIM"), ("defi", "ORPHA"), (None, "ORPHA")],
+)
+def test_query_omim(adapter, test_omim_term, test_orpha_term, source, query):
```

Comment:
Rename the test so it's reflecting the renamed endpoint (it's returning all terms, not just omim)

Reply:
It has been updated accordingly when renaming the function